docs/modules/ingest/pages/overview.adoc

```diff
@@ -115,6 +115,10 @@ With MapStore, you can do the following:
 - Hydrate the cache upon startup to prevent poor performance caused by many cache misses.
 - Pre-process or enrich data in real time before ingesting it by writing custom Java code.
 
+=== Read-Through Caching Only
+
+If you only need to ingest data and cache it in Hazelcast, consider using the xref:mapstore:configuring-a-generic-maploader.adoc[generic MapLoader].  
```

Comment:
There is a difference between `ingest & cache data` and `read-through caching`, maybe we should clarify somewhere in this page?

Reply:
Yes, that's correct.

MapLoader/Store is for read-through/write-through caching.

If you want one-time load you shouldn't use map loader at all, but use Jet pipeline / SQL instead.

Reply:
Updated as requested.